### ISSUE TITLE
How to reduce log level to warning from info

### ISSUE THREAD

**Issue by Mahesh-Gunda-Maersk** [mittwald/kubernetes-replicator#269]
**Describe the bug**
right now, replicator logs show log levels as info and logging every 30 mins. we would like to reduce the log level to a warning as we fwd logs to the Central Monitoring system. how do we achieve that? 
Couldn't find options in the helm chart.  Please share if any args.  we can change the log level to info or debug whenever we wanted to. 

**Example Log Trace**
```
time="2023-04-17T14:08:25Z" level=info msg="Checking if kube-public/cluster-tls-certificate exists? true" kind=Secret source=hybrid-cloud/cluster-tls-certificate target=kube-public/cluster-tls-certificate
```
**Expected behavior**
Log only when error not everything. 

**Environment:**
 - Kubernetes version: [1.25]
 - kubernetes-replicator version: [v2.7.3]

**Additional context**

**Comment by martin-helmich** [comment]
The replicator supports a `--log-level` flag, which you should be able to use.

https://github.com/mittwald/kubernetes-replicator/blob/902b3c3cb2e6426da7f01860b805225a4b3f9a5e/main.go#L30

The Helm chart has an `args` value which can be used to pass custom command-line arguments to the Deployment. Something like the following in your Helm values should do the trick:

```yaml
args:
  - --log-level=warn
```

**Comment by Mahesh-Gunda-Maersk** [comment]
ok. was looking for something like this. will test this. Thank you.

**Comment by Mahesh-Gunda-Maersk** [comment]
Though I set log-level as warn, the log level is showing as an error ..  is this expected?  also when I set it to trace I see both debug, info & error logs.  Please help understand this behaviour

![image](https://user-images.githubusercontent.com/111576197/232984376-8e8811a9-a529-4153-a5e3-2f1759e32479.png)
![image](https://user-images.githubusercontent.com/111576197/232985418-09156c27-afd0-4657-89bd-02dc2a247ee3.png)

**Comment by Mahesh-Gunda-Maersk** [comment]
Hi, pls check if this behavior is fine.  Thank you.

**Comment by martin-helmich** [comment]
> Though I set log-level as warn, the log level is showing as an error .. is this expected? also when I set it to trace I see both debug, info & error logs. Please help us in understanding this behaviour

I'm unsure if I understand the issue. Setting a log level customarily results in logging items with a severity equal to OR HIGHER than the configured log level, so a log level of `warn` will also log anything more severe than warnings (like errors and panics).

**Comment by Mahesh-Gunda-Maersk** [comment]
yes, we are on the same page.  I have set log-level to the warning,  but I see logging severity as error.   just want to understand if this behavior is expected. anything we need to worry about?

**Comment by martin-helmich** [comment]
Alright, then. 🙂  Regarding the actual logs with error severity: No, in normal operation, the replicator should not log any errors. What is the actual error that is being reported? I'm referring to the `error` field that is cropped in your screenshot.

**Comment by Mahesh-Gunda-Maersk** [comment]
below is the log snippet with log-level=warn


`time="2023-04-19T06:13:15Z" level=error msg="could not replicate object to other namespaces" error="Replicated xyz-streams/local-abc-configuration to 7 out of 17 namespaces: 10 errors occurred:\n\t* Failed to replicate Secret xyzl-streams/local-abc-configuration -> kube-public: Failed to update secret kube-public/local-abc-configuration: secrets \"local-kafka-configuration\" already exists: Failed to update secret kube-public/local-abc-configuration: secrets \"local-kafka-configuration\" already exists\n\t ...........................`

same error message for other secrets.

**Comment by Mahesh-Gunda-Maersk** [comment]
Hello @martin-helmich, were you able to reproduce this issue at your end?  hope the error snippet shared helped.

**Comment by Mahesh-Gunda-Maersk** [comment]
sorry to bother you @martin-helmich.  did you get a chance to look at the log snippet and do you have anything to comment or advise on this behavior?
